frontends/verilog/verilog_parser.y

```diff
@@ -643,7 +643,25 @@ wire_type_token_io:
 		astbuf3->is_output = true;
 	};
 
+wire_type_signed:
+	TOK_SIGNED   { astbuf3->is_signed = true;  } |
+	TOK_UNSIGNED { astbuf3->is_signed = false; } |
+	%empty;
+
+wire_type_const_rand:
+	TOK_CONST { current_wire_const = true; } |
+	TOK_RAND { current_wire_rand = true; } |
```

Comment:
Can we use the same extra alignment spacing as in the above rule?

Reply:
Fixed.